src/App.jsx

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
-// Views
-import LandingPage from './views/LandingPage';
-import ProfilePage from './views/ProfilePage';
+//Views
+import LandingPage from './views/LandingPage.jsx';
+import ProfilePage from './views/ProfilePage.jsx';
```

Comment:
Is there a reason you specified the file format `.jsx`?, kindly confirm this.

Reply:
Thank you for pointing this out. I'll rectify now.
I deleted the lines without the extension instead of deleting these lines with extension.
It was an oversight on my part, I was particular about this line ```import './assets/scss/style.scss';```